kura/org.eclipse.kura.wire.helper.provider/about_files/epl-v10.html

```diff
@@ -0,0 +1,261 @@
+<?xml version="1.0" encoding="ISO-8859-1" ?>
```

Comment:
Every bundle in Kura contains this license file and that's the reason, it has been kept here as well.